packagebuild/daisy_startupscript_rpm.sh

```diff
@@ -75,6 +77,7 @@ fi
 try_command yum install -y $GIT rpmdevtools yum-utils python3-devel
 
 git_checkout "$REPO_OWNER" "$REPO_NAME" "$GIT_REF"
+ROOT_WORK_DIR=$(pwd)
```

Comment:
missed earlier, don't we need to install protoc?

Reply:
yep, I missed that too. I'll get it done.

Reply:
done. thank you.